mmv1/products/firebase/api.yaml

```diff
@@ -132,6 +134,16 @@ objects:
       error: !ruby/object:Api::OpAsync::Error
         path: 'error'
         message: 'message'
+    parameters:
+      - !ruby/object:Api::Type::String
+        name: 'deletion_policy'
+        description: |
+          Optional. Set to `ABANDON` to soft-delete the WebApp. Soft-deleted WebApps are recoverable
+          within 30 days using [projects.webApps.undelete](https://firebase.google.com/docs/projects/api/reference/rest/v1beta1/projects.webApps/undelete).
+          Set to `DELETE` to immediately delete the WebApp. Default to `DELETE`
+        input: true
+        url_param_only: true
+        default_value: DELETE
```

Comment:
This is a pretty significant change. The resource behavior is going from un-delete-able to default delete-able. Should the default be abandon with delete being set within our tests?

Reply:
What are the other valid options for this?